src/habitsToTaskRouter.ts

```diff
@@ -0,0 +1,45 @@
+import express from 'express';
+import axios from 'axios';
+import * as CONSTANTS from '../lib/constants';
+import { getDateFormatted } from '../lib/utils';
+
+const router = express.Router();
+
+router.post('/habit-as-task', async (req, res) => {
+  try {
+    const { parentId, timeEstimate, title, record } = req.body
+    if (parentId === CONSTANTS.UNASSIGNED_PARENT_ID) {
+      return res.status(200).json({ message: `Skipping creating a task for habit with name ${title}` })
```

Comment:
Could move all the messages into a constants file as the number of messages grow (future refactor)

Reply:
Ack, added note to project tracker